bofire/data_models/base.py

```diff
@@ -12,6 +13,8 @@ class Config:
         validate_assignment = True
         arbitrary_types_allowed = False
         copy_on_model_validation = "none"
+        extra = Extra.forbid
```

Comment:
please remove the lines above that contain the TODO to actually do this :-D

Reply:
Thanks for the hint. Done!